### PR TITLE
Fix datasets

### LINKED ISSUE #14: Add introduction downloaded the data to the relevant episodes
- [x] Add data description in the index page
- [x] Add data downloading instrution in the setup page
- [ ] In each episode refer to the index page for data description
- [x] all Eps: use data in the `data` folder

### PR DESCRIPTION
**Pull request by fnattino**
Closes #14 . I have made it such that all files generated by the lesson are stored in the current working directory, while the input vector files are in the `data` directory. Not sure whether this is the best, but it might simplifies the life of participants tracking what needs to be loaded in, and it would be consistent with the best practices taught by other Carpentries' lessons where you should not save output in the same folder as input..

### CODE REVIEW

**Comment by fnattino** [comment]
@rogerkuou , still missing the callouts within each chapter, but you can go ahead and review what is there so far

File: _episodes/07-vector-data-in-python.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ objectives:
 - "Access the attributes of a spatial object with `geopandas`."
 keypoints:
 - "Vector dataset metadata include geometry type, CRS, and extent."
-- "Load spatial objects into Python with the `geopandas.read_file()` method."
-- "Spatial objects can be plotted directly with `geopandas.GeoDataFrame.plot()`."
+- "Load spatial objects into Python with `geopandas`'s `read_file()` function."
+- "Spatial objects can be plotted directly with `GeoDataFrame`'s `.plot()` method."
```
Comment (rogerkuou) [inline _episodes/07-vector-data-in-python.md:14]: I found this format:  `geopandas`'s `read_file()` a bit weird. Can we maybe use "the `geopandas.read_file() function`" instead?

**Review by rogerkuou — COMMENTED** [review]
Thx @fnattino for correcting the vector exercise! just one minor comment.

Also, just to check, are we following the convention of saving the downloaded file in `data`, and exporting data to the working directory?

File: _episodes/07-vector-data-in-python.md (hunk 1 above)
Comment (fnattino) [reply _episodes/07-vector-data-in-python.md:14]: You are right - it was just that `geopandas` is always imported as `gpd` in the episode, but I am sure it is clear enough.

**Comment by fnattino** [comment]
> Also, just to check, are we following the convention of saving the downloaded file in `data`, and exporting data to the working directory?

Yes, this is what I thought would be best (everything that is saved from within the lesson is saved in the current working directory to avoid mixing it with the input files).

**Comment by fnattino** [comment]
Hi @rogerkuou, callouts to introduce the data (and the alternative download method) have been introduced, can you have a look? When this and #59  are merged we can then open the big upstream PR

**Review by rogerkuou — APPROVED** [review]
@fnattino Looks good! I do not have further comments. Will merge this,

**Pull request merged**